sdk/simulation/urtssim/enclave_creator_sim.cpp

```diff
@@ -118,30 +108,22 @@ int EnclaveCreatorSim::create_enclave(secs_t *secs, sgx_enclave_id_t *enclave_id
         LockGuard lock(&s_enclave_info_mutex);
         if (s_enclave_elrange_map.count(base_address) != 0)
```

Comment:
Since the values aren't heap-allocated, and s_enclave_elrange_map[base_address] will create a new entry if it didn't exist, you don't need the two cases.